README.md

```diff
@@ -7,6 +7,26 @@
 
 In this work we present a semantic data model for [the set of common data elements for rare diseases registration](https://eu-rd-platform.jrc.ec.europa.eu/sites/default/files/CDS/EU_RD_Platform_CDS_Final.pdf) recommended by the European commission Joint Research Centre. There are 16 data elements: ‘Pseudonym’, ‘Date of Birth’, ‘Sex’, ‘Patient’s status’, ‘Date of death’, ‘First contact with specialised centre’, ‘Age at onset’, Age at diagnosis’, ‘Diagnosis of the rare disease’, ‘Genetic diagnosis’, ‘Undiagnosed case’, ‘Agreement to be contacted for research purposes’, ‘Consent to the reuse of data’, ’Biological sample’, ‘Link to a biobank’, ‘Classification of functioning/disability’. We proposed a sematic data model for each of these data elements.
 
-### To learn more
+## CDE modules overview
 
-To know more about our work please see the [wiki page](https://github.com/ejp-rd-vp/ERN-common-data-elements/wiki) of this repository.
+The figure below gives an overview of upper level concepts and properties used in our cde models.
```

Comment:
You said "we present a semantic data model" so I think "our cde models" ---> "our CDE model".